framework/integration/test/basic/basic_test.go

```diff
@@ -8,14 +8,19 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/giantswarm/operatorkit/framework/integration/client"
+	"github.com/giantswarm/operatorkit/framework/integration/wrapper"
+	"github.com/giantswarm/operatorkit/framework/integration/wrapper/configmap"
 )
 
 // Test_Finalizer_Integration_Basic is a integration test for basic finalizer
 // operations. The test verifies that finalizers are added and removed as
 // expected. It does not cover correct behavior with reconciliation.
+//
+// !!! This test does not work with CRs, the framework is not booted !!!
+//
 func Test_Finalizer_Integration_Basic(t *testing.T) {
```

Comment:
I think this can be table driven test where you pass different crud and object.

Reply:
This was the idea here, but the basic test can *not* run with a CR because the CRD is never ensured if the framework is not started. I agree on the reconciliation test.

Reply:
So you can add `init func() error` to the test struct.

Reply:
I don't follow or understand what you mean here. What are you trying to fix?

Reply:
I mean you can create the CR in the init function. You can use 
 CRDClient https://github.com/giantswarm/operatorkit/blob/c656522a5b6cf0c811f4d4ba408306b49121c298/client/k8scrdclient/k8scrdclient.go#L20 for that. It's idempotent.

Reply:
Oh, okay - got it! Can I do this in a separate PR? I don't want to make this one even bigger.